job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/workflow/OrganizationFolderTriggerContext.groovy

```diff
@@ -0,0 +1,59 @@
+package javaposse.jobdsl.dsl.helpers.workflow
+
+import javaposse.jobdsl.dsl.Item
+import javaposse.jobdsl.dsl.JobManagement
+import javaposse.jobdsl.dsl.helpers.triggers.ItemTriggerContext
+
+/**
+ * @since 1.56
+ */
+class OrganizationFolderTriggerContext extends ItemTriggerContext {
+
+  OrganizationFolderTriggerContext(JobManagement jobManagement, Item item) {
+    super(jobManagement, item)
+  }
+
+  /**
+   * Sets the periodic trigger for when this has not been triggered.
+   * @since 1.56
+   */
+  void periodicIfNotOtherwiseTriggered(PeriodicFolderTrigger periodicFolderTrigger) {
+    triggerNodes << new NodeBuilder().'com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger' {
```

Comment:
Open question: the [`MultibranchWorkflowTriggerContext`](https://github.com/jenkinsci/job-dsl-plugin/blob/master/job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/MultibranchWorkflowTriggerContext.groovy#L18-L23) has a different pattern, but the UI presents a dropdown here. What is the best option to go with?

At least from my standpoint, we switched from `Multibranch` to this SCM Navigator style almost entirely, so consistency may matter for the Job DSL.